include/swift/ABI/MetadataValues.h

```diff
@@ -643,7 +643,8 @@ class TargetParameterTypeFlags {
   enum : int_type {
     InOutMask    = 1 << 0,
     SharedMask   = 1 << 1,
-    VariadicMask = 1 << 2,
+    OwnedMask    = 1 << 2,
```

Comment:
I don't think we need both `shared` and `owned` here, since `owned` is the opposite of `shared`.  But actually, I think you ought to restructure this so that there's a 2-bit ownership field that just stores a ValueOwnership value.

Although giving it 2 bits seems unwisely miserly when we already have two values.   In practice, ParameterTypeFlags is 32/64 bits, right?  We could give it... well, at least 4.